catalog/region-specific-policies.yaml

```diff
@@ -0,0 +1,53 @@
+# https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples.html#examples_general
+- sid: "DenyRegionUsage"
+  effect: "Deny"
+  not_actions:
```

Comment:
"Deny" with `not_actions` - does it mean to allow everything that is not in the list?

Reply:
base on the docs 
```This policy uses the Deny effect to deny access to all requests for operations that don't target one of the two approved regions (eu-central-1 and eu-west-1). The [NotAction](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_notaction.html) element enables you to list services whose operations (or individual operations) are exempted from this restriction. Because global services have endpoints that are physically hosted by the us-east-1 Region , they must be exempted in this way. With an SCP structured this way, requests made to global services in the us-east-1 Region are allowed if the requested service is included in the NotAction element. Any other requests to services in the us-east-1 Region are denied by this example policy.
```